packages/backend/package.json

```diff
@@ -1,19 +1,27 @@
 {
   "name": "example-backend",
   "version": "0.1.1-alpha.12",
-  "main": "dist/index.cjs.js",
+  "main": "src/index.ts",
```

Comment:
Whoa!

Reply:
Realized we shouldn't change this or the publishConfig, did it because we were requiring the postpack script. Got rid of it and removed the postpack requirement instead.